src/app/main-nav/main-nav.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { menuItems } from "../mat-main-nav/menu-items.const";
+import { IMenuItem } from "../mat-main-nav/menu-item.interface";
+import { environment } from "@environment";
+
+@Component({
+  selector: "app-main-nav",
+  templateUrl: "./main-nav.component.html",
+  styleUrls: ["./main-nav.component.scss"]
+})
+export class MainNavComponent implements OnInit {
+  @Input() skipLinkSelector: string = "maincontent";
+  showMenu: boolean = false;
```

Comment:
could you please run linting on this pr as there's a couple of variables that are automagically trivially inferred by there literal values

Reply:
done